### DISCUSSION TITLE
Sensor parameter not recognized.

### DISCUSSION THREAD

**Discussion by ajkelsey** [raspberrypi/picamera2#948] [Q&A]
I am trying to use the sensor parameter in the camera configuration and I am receiving a type error. Any ideas what the issue could be?

![image](https://github.com/raspberrypi/picamera2/assets/64913190/cfdffc59-afe4-40c2-b32f-e3d898f79e1c)

**Comment by davidplowman** [comment]
It sounds like you may have updated Picamera2 but not libcamera. In fact, some Pis do not support the latest Picamera2 versions because they are running Bullseye which no longer receives libcamera updates. The best advice, if this possible for you, is always to install using apt. This will always provide versions that work together.
